cypress/e2e/createEvent.cy.js

```diff
@@ -0,0 +1,49 @@
+describe('Event Info', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.getEventOne();
+      cy.clickEventOne();
+      cy.intercept('GET', 'https://lynk-up-server.onrender.com/users/888-888-8888', {
+        fixture: 'getUser.json'
+      })
+      cy.intercept('GET', 'https://lynk-up-server.onrender.com/events/1', {
+        fixture: 'event1.json'
+      })
```

Comment:
Can remove all of these - unnecessary for these tests and/or repeats of the `cy.login()` functionality